src/radix_connect/mobile/relay_service/models/request/request.rs

```diff
@@ -0,0 +1,306 @@
+use super::request_method::Method;
+use crate::prelude::*;
+use crate::radix_connect::mobile::session::session_id::SessionID;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Request {
+    pub method: Method,
+    pub session_id: SessionID,
+    pub data: Option<BagOfBytes>,
+}
+
+impl Request {
+    pub fn new(
+        method: Method,
+        session_id: impl Into<SessionID>,
+        data: impl Into<Option<BagOfBytes>>,
+    ) -> Self {
+        Self {
+            method,
+            session_id: session_id.into(),
+            data: data.into(),
+        }
+    }
+}
+
+impl Request {
+    pub fn new_send_request(
```

Comment:
I think we should rename this just `send_request`, that is the naming convention I use elsewhere, see e.g. `NonFungibleLocalId` (ops it should have been named `LocalID` my bad!!):
https://github.com/radixdlt/sargon/blob/develop/src/profile/v100/address/non_fungible_local_id.rs#L27-L45

Reply:
I just wanted to group together the constructors with the `new` prefix

Reply:
understood, but I follow a pattern which RET and Scrypto also follows, so I think best not use `new`. 

(Juxtaposition: however, for UniFFI exported global functions which are "constructors" we *always* use `new_name_of_type`)

Reply:
@GhenadieVP oh I realized I was in the wrong! Lets DO USE `new_` because this is a `struct`! Not an enum! :) and lets use the name of the variant for enums only... sorry! :D